translations/et.php

```diff
@@ -48,27 +48,36 @@
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_93a524d9a34a84b2c8b9f0575c08b6c8'] = 'Loomisaeg';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_9c34a97f10f573b8d1cc54832a13da30'] = 'Viimati uuendatud';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_49ee3087348e8d44e1feda1917443987'] = 'Nimi';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_3601146c4e948c32b6424d2c0a7f0118'] = 'Hind';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_97f08a40f22a625d0cbfe03db3349108'] = 'Toote ID';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_1ef176547b783195eb9c06166dba55e6'] = 'Järjestamise suund';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_cf3fb1ff52ea1eed3347ac5401ee7f0c'] = 'Kasvav';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_e3cf5ac19407b1a62c6fccaff675a53b'] = 'Kahanev';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_fe760630d69004060a872c86c116d120'] = 'RSS-voog';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_configure_e046e1a272245a642b1a3ee0b7b45e1d'] = 'Kopeeri see URL oma kirjapõhja väljale';
 $_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_3805f49499fa5010c394e219aa1fe7a0'] = 'Saage meie värskeimaid uudiseid ja eripakkumisi';
 $_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_a4d3b161ce1309df1c4e25df28694b7b'] = 'Telli';
 $_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_198584454b0ce1101ff5b50323325aa8'] = 'Teie e-posti aadress';
 $_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_e3bf47a46c6efd3bb53fedb5136bdf1c'] = 'Te võite igal ajal oma uudiskirja tellimuse tühistada.';
 $_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_cbef6aeeb39bd1078e8869492fd83f1c'] = 'Täname uudiskirjaga liitumise eest!';
 $_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_e3678541149247bb751545e796269b8f'] = 'Midagi läks valesti uudiskirjaga liitumisel, palun proovi uuesti!';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_column_3805f49499fa5010c394e219aa1fe7a0'] = 'Saage meie värskeimaid uudiseid ja eripakkumisi';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_column_a4d3b161ce1309df1c4e25df28694b7b'] = 'Telli';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_column_198584454b0ce1101ff5b50323325aa8'] = 'Teie e-posti aadress';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_column_e3bf47a46c6efd3bb53fedb5136bdf1c'] = 'Te võite igal ajal oma uudiskirja tellimuse tühistada.';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_column_cbef6aeeb39bd1078e8869492fd83f1c'] = 'Täname uudiskirjaga liitumise eest!';
+$_MODULE['<{smailyforprestashop}prestashop>smaily_blocknewsletter_column_e3678541149247bb751545e796269b8f'] = 'Midagi läks valesti uudiskirjaga liitumisel, palun proovi uuesti!';
```

Comment:
These seem to be exactly the same. Are both needed?

Reply:
Prestashop automatically added them. I removed them manually, not sure why this happened